client/src/use/useEventChart.ts

```diff
@@ -26,10 +27,12 @@ export default function useEventChart({
     const mapfunc = typeStyling.value.color;
     const selectedTrackIdValue = selectedTrackId.value;
     /* use forEach rather than filter().map() to save an interation */
-    enabledTracks.value.forEach((track) => {
+    enabledTracks.value.forEach((filtered) => {
+      const { track } = filtered;
       const { confidencePairs } = track;
       if (confidencePairs.length) {
-        const trackType = confidencePairs[0][0];
+        const type = track.getType(filtered.context.confidencePairIndex);
+        const trackType = type ? type[0] : '';
```

Comment:
This is causing me to question the signature of `track.getType()`

It should probably always return `[string, number]` instead of the possibility of a null type, right?  Like, if you ask for an out-of-bounds pair, that seems like an invalid execution path that should also throw an error from within the getter.

Why would `getType` ever be null?  Why would you ever have a reference to a confidencePair index that doesn't exist?